lib/lotus/entity.rb

```diff
@@ -138,6 +138,17 @@ module ClassMethods
       #
       #   User.attributes => #<Set: {:id, :name}>
       #   DeletedUser.attributes => #<Set: {:id, :name, :deleted_at}>
+      #
+      # @example Add timestamps
+      #   require 'lotus/model'
+      #
+      #   class User
+      #     include Lotus::Entity
+      #     attributes :timestamps
```

Comment:
Why do we have 2 different ways in defining timestamp? For repo, we do 		+      #     include Lotus::Repository::Timestamps
, why don't we do the same here?